src/main/java/seedu/duke/food/Food.java

```diff
@@ -141,6 +138,13 @@ public Food(String name, String expiryDate, Double quantity, String unit, String
         this.unit = unit;
     }
 
+    public Food(String name, String expiryDate, String category) {
+        this.name = name;
+        this.expiryDate = expiryDate;
+        this.category = FoodCategory.valueOf(category);
+    }
+
+
```

Comment:
Hey @ZhongXiangWong , think you missed to add the validation part in the Food constructor, can you put it there?

Reply:
Hi, I feel that the validator method is not very suitable for use in this case as it throws an exception which is not useful in this case. I feel that a boolean method which return `true\false` will be much better I can set the category to the default category `unclassified_food` as I cannot prompt the text file to give me another value.